src/clj/game/cards/assets.clj

```diff
@@ -88,6 +88,35 @@
                                                :msg (msg "trash " (join ", " (map :title targets)))))]
                              (continue-ability state side ab agg nil)))})
 
+   "Alexa Belsky"
+   {:abilities [{:label "[Trash]: Shuffle all cards in HQ into R&D"
+                 :effect (req (trash state side card)
+                              (if (pos? (:credit runner))
```

Comment:
The runner could use credits on Net Mercur for this, hence could prevent cards even on 0 credits.